pkg/policy/mapstate.go

```diff
@@ -100,7 +134,12 @@ func (keys MapState) RedirectPreferredInsert(key Key, entry MapStateEntry) {
 // endpoint.
 func (keys MapState) DetermineAllowLocalhostIngress(l4Policy *L4Policy) {
 	if option.Config.AlwaysAllowLocalhost() {
-		keys[localHostKey] = MapStateEntry{}
+		keys[localHostKey] = MapStateEntry{
```

Comment:
nit: This should use the function above too, so that if it gets refactored further in future the compiler will point out any further required changes, rather than silently ignoring / zeroing the added fields.

Reply:
(Same for the other direct initializations)